roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -198,3 +198,26 @@
   when: "'postfix' in ansible_facts.packages"
   changed_when: "task.rc == 2"  # rc == 2 => file doesn't exist
   failed_when: "task.rc == 1"  # rc == 1 => postalias failed
+
+- <<: *task
+  vars:
+    taskid: 10-067
+    name: "Netboxinventory: Gather pkg list and output to json"
+  ansible.builtin.package_facts:
+    manager: auto
+
+- <<: *task
+  vars:
+    taskid: 10-067
+    name: "Netboxinventory: Gather pkg list and output to json"
+  ansible.builtin.lineinfile:
+    create: yes
+    mode: 0744
+    path: "host_packages_collection-{{ ansible_date_time.date | string }}.txt"
+    insertafter: EOF
+    line: |
+      {"Host": [
+      {"Hostname":  "{{ inventory_hostname }}"},
+      {"OS": "{{ hostvars[inventory_hostname].ansible_distribution }} {{ hostvars[inventory_hostname].ansible_distribution_version }}" },
+      {"Package list update": "{{ ansible_date_time.iso8601 }}", "Package list": [ {{ ansible_facts.packages | to_json }} ] } ] }
+  delegate_to: 127.0.0.1
```

Comment:
Please use `localhost` rather than `127.0.0.1`, as `localhost` is a special hostname for Ansible that connects to the Ansible controller using `ansible_connection=local`, whereas if you use `127.0.0.1`, Ansible tries to use SSH to localhost.

Also, please put the `deletate_to` line` above the `lineinfile` line, as it's easy to miss at the bottom.

Reply:
Fixed :+1: